homeassistant/components/zwave_js/update.py

```diff
@@ -183,17 +200,17 @@ async def async_release_notes(self) -> str | None:
             return None
         return self._latest_version_firmware.changelog
 
-    async def async_install(
-        self, version: str | None, backup: bool, **kwargs: Any
-    ) -> None:
+    async def async_install(self, *_: Any, **__: Any) -> None:
         """Install an update."""
         firmware = self._latest_version_firmware
         assert firmware
-        self._attr_in_progress = 0
-        self.async_write_ha_state()
+        self._reset_progress(True)
         self._progress_unsub = self.node.on(
             "firmware update progress", self._update_progress
         )
+        self._finished_unsub = self.node.once(
```

Comment:
We never use `self._finished_unsub`.

Reply:
thanks, do we need it for unload or should I just schedule it and be done with it?